src/response.rs

```diff
@@ -650,6 +747,7 @@ impl Clone for Response {
             ext: Extensions::new(),
             peer_addr: self.peer_addr.clone(),
             local_addr: self.local_addr.clone(),
+            error: None,
```

Comment:
Is this because we don't have a Clone bound on the error? This seems like a source of confusion

Reply:
Yeah, I don't really know how to Clone `anyhow::Error` /  `std::error:Error` correctly.

Reply:
I guess I can go back to storing it in an `Rc` (or an `Arc`? Idk how much thread-safety is involved here) and then return a reference out of the `Rc` on `error()`...

Reply:
That seems like a possibility, but I'm not sure who uses Response::clone or how important it is. I don't think tide or surf clone a response as part of normal behavior, so maybe it's just for testing?